src/common-utils/devcontainer-feature.json

```diff
@@ -40,7 +40,7 @@
         "userUid": {
             "type": "string",
             "proposals": [
-                "1000",
+                "1001",
```

Comment:
Can we update the patch version? Thanks! 

https://github.com/devcontainers/features/blob/4c7261225d5674773cf7f7bdebacf8dffbaf1a13/src/common-utils/devcontainer-feature.json#L3

Reply:
I created a pull request #539.

Reply:
It might be worth adding a `modifyUsername` parameter so that users can update the username instead of assume the intention is to create a new user when the username doesn't match the username of the base image.

Reply:
maybe it would be better to have added 1001 instead of modify the option... If you want I can add the original back as another option, an addition instead of substitution, so that the full list of options demonstrate the available capabilities of the feature.